### ISSUE TITLE
2 feature requests

### ISSUE THREAD

**Issue by abdcef** [alangrainger/obsidian-google-photos#7]
Thank you for this useful plugin!

- Is there a way to include photo timestamps when adding them in Obsidian?
- Can I add multiple photos at the same time? An option to automatically add all photos for a particular date to that daily note will be helpful.

**Comment by alangrainger** [comment]
Can you expand what you mean by point #1? The filenames are by default based off the photo date-taken timestamp, and the format is configurable.

**Comment by abdcef** [comment]
Thanks, it is good to see the timestamp included.
Can I change the pasted text to something like the following:
[![](file name)(online link)] hh:mm AM/PM

**Comment by alangrainger** [comment]
Request 1 sounds good, I'll add that in with the ability to completely customise the inserted text using your own format and placeholders.

Request 2 I won't do any time soon, but you're welcome to work on it yourself and submit a pull request.

And before you say you can't code - neither could I before starting these kind of personal projects. I watched a course on Udemy and just gave it a go.

**Comment by alangrainger** [comment]
Timestamps have been added in v1.2.3.

Set your custom Markdown template to be something like this:

```
[![]({{ local_thumbnail_link}})]({{google_photo_url}}) {{taken_date}} 
```

At the moment you can't customise the date/time format, but the current inserted format is the ATOM date format for maximum compatibility.

**Comment by ashishbhateja** [comment]
Thank you!

**Comment by abdcef** [comment]
Thanks for the added features! 
I wonder if I can use ![](google_base_url)? Then no need for a thumbnail to be downloaded into my vault.
